my-app/pages/index.js

```diff
@@ -33,6 +33,14 @@ export default function Home() {
     // Connect to Metamask
     // Since we store `web3Modal` as a reference, we need to access the `current` value to get access to the underlying object
     const provider = await web3ModalRef.current.connect();
+    // Subscribe to accounts change
```

Comment:
can you add your discord id as well? We can give you the early contributor role on discord?

Reply:
discord id: mmblack#6621